src/api/controllers/members.ts

```diff
@@ -62,13 +62,14 @@ export async function handleInternalGuildMemberUpdate(data: DiscordPayload) {
     deaf: guildMember?.deaf || false,
     mute: guildMember?.mute || false,
     roles: payload.roles,
+    nick: payload.nick === undefined ? null : payload.nick,
   };
   const member = await structures.createMember(
     newMemberData,
     payload.guild_id,
   );
 
-  if (guildMember?.nick !== payload.nick) {
+  if (guildMember?.nick && payload.nick && guildMember.nick !== payload.nick) {
```

Comment:
Irrelevant change; please remove it and create a new PR or should I do it?

Reply:
This change is needed

Reply:
But I can just add `!` too.
```ts
if (guildMember?.nick !== payload.nick) {
  eventHandlers.nicknameUpdate?.(
    guild,
    member,
    payload.nick!,
    guildMember?.nick!,
  );
}
```

Reply:
You can create a PR with base as `next` and once that PR is merged to the `next` branch, you may fetch the latest `next` to this branch and you will have updated code. :)